### ISSUE TITLE
libcurl-dev Required

### ISSUE THREAD

**Issue by withmetta** [GaloisInc/curl#20]
So I tried running "cabal install curl" which crashed as I needed liburl4-openssl-dev or one of the other alternative versions. IDK if this fail more gracefully without having to dig through the error log and then Googling, but I thought I should at least ask. 

Maybe this isn't even an issue and I'm just new to the space, but I thought I should at least make a post in case.

**Comment by glguy** [comment]
No, there's not much to do here. To install the bindings for a C library you have to have the C library installed.
